store/prefix/store.go

```diff
@@ -1,207 +0,0 @@
-package prefix
-
-import (
-	"bytes"
-	"errors"
-	"io"
-
-	"cosmossdk.io/store/cachekv"
-	"cosmossdk.io/store/tracekv"
-	"cosmossdk.io/store/types"
-)
-
-var _ types.KVStore = Store{}
-
-// Store is similar with cometbft/cometbft/libs/db/prefix_db
-// both gives access only to the limited subset of the store
-// for convinience or safety
-type Store struct {
```

Comment:
we should add this to the tracking issue if we need to bring it back or apps will import v1 and v2 of store